.github/workflows/lint.yml

```diff
@@ -6,6 +6,15 @@ on:
   pull_request:
 
 jobs:
+  actionlint:
```

Comment:
I'm realizing now that it's too bad that there's no way to have a linter that runs locally (I don't like things that require a PR to validate / can't be perfected and checked by local tooling without having to generate repo noise), but this is still an improvement and it doesn't seem like there are comparable local tools out there!

Reply:
Oh, I meant to document this in the README - you can run it locally! Go to https://github.com/rhysd/actionlint and either compile it locally or grab a release binary. I'll document that before merging.